src/AltinnCore/Designer/Controllers/RulesController.cs

```diff
@@ -295,16 +310,15 @@ public IActionResult GetFile(string org, string service, string name)
         [HttpPost]
         public IActionResult SaveImplementationFile(string org, string service, string fileName, string fileContent)
         {
-          if (fileName == "RuleHandler.js")
-          {
-            _repository.SaveResourceFile(org, service, fileName, fileContent);
-          }
-          else
-          {
-            _repository.SaveImplementationFile(org, service, fileName, fileContent);
-          }
-
-          return StatusCode(200);
+            if (fileName == "RuleHandler.js")
+            {
+                _repository.SaveResourceFile(org, service, fileName, fileContent);
+            }
+            else
+            {
+                _repository.SaveImplementationFile(org, service, fileName, fileContent);
+            }
```

Comment:
Stylecop fails, need to ad a blank line:  error SA1513: Closing brace should be followed by blank line****